lm-agent/lm_agent/services/clean_jobs_and_bookings.py

```diff
@@ -67,7 +67,7 @@ def extract_usages_from_report(report_item: LicenseReportItem) -> List[Extracted
         ExtractedUsageSchema(
             feature_id=report_item.feature_id,
             username=usage["username"],
-            lead_host=usage["lead_host"],
+            lead_host=usage["lead_host"].split(".")[0],
```

Comment:
Should probably add a comment explaining why this logic is being applied here.

Reply:
Done!